.azure-pipelines/client.yml

```diff
@@ -34,6 +34,12 @@ jobs:
       scriptPath: 'scripts/devops_tasks/build_packages.py'
       arguments: '-d "$(Build.ArtifactStagingDirectory)" "$(build_targeting_string)"'
 
+  - bash: |
```

Comment:
If we can whittle this down to just the single "twine check" line, we can then change this task type to "script"

Reply:
Done.